src/cable/base/cable_base_addr.cpp

```diff
@@ -23,10 +23,13 @@ unique_ptr<c_cable_base_addr> c_cable_base_addr::cable_make_addr(const string &
 			_try_user(pos1>0); // string::substr is safe, but anyway test against user doing --peer "-foo"
 			string part1 = str.substr(0,pos1);
 			string part2 = str.substr(pos1+1);
-			if (part1=="auto" || part1=="udp" || part1=="tcp") { //udp for those types of cables
+			if (part1=="auto" || part1=="udp") { //udp for those types of cables
```

Comment:
comment: udp=>tcp

Reply:
hm? We just support "udp:" cable protocol.  TCP will be added later, so removed it for now (anyway it would be separate if expression of course)

Reply:
OK